src/components/10-atoms/input-text/index.js

```diff
@@ -244,11 +251,13 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
       placeholder = '',
       disabled,
       isReact,
-      maxLength,
+      maxLength = '',
       invalid,
       checkMark,
       isControlled,
       refId,
+      inputmode = '',
+      pattern = PATTERN_DEFAULT, // if we do not set a default value here the result on DOM is pattern="undefined", that causes problems on validating a form
```

Comment:
What about an empty string, like for `inputmode`?

Reply:
the value for pattern have to be a "RegEx". And an empty string is not a good RegEx because anything but empty string is matching --> and on submitting a form this errormessage occurs (on chrome).